src/components/WebexCreateSpace/WebexCreateSpace.jsx

```diff
@@ -62,17 +65,50 @@ export default function WebexCreateSpace({
   };
 
   const onError = (error) => {
+    const endTime = window.performance.now();
+
+    emitMetrics({
+      fields: {
+        startTime,
+        endTime,
+        totalTime: endTime - startTime,
+      },
+      metricName: 'createSpaceTime',
```

Comment:
Can we have the name as createSpaceError and createSpaceSuccess?

Reply:
Done.